R/03-geojob-obj.R

```diff
@@ -43,19 +45,53 @@ setMethod(f="initialize",signature="geojob",
 #' @author Jordan S Read
 #' @rdname geojob-methods
 #' @export
-setGeneric("geojob", function(...) {
+setGeneric("geojob", function(xml, ...) {
```

Comment:
I think having this parameter be more generic (in R it might be common to see it as `x`, `object`, `.Object`, etc) would help if there were other methods to dispatch to when the value of `xml` is something other than actual xml (for example, it may be a `geojob` or an `id` or a url of some kind)